File: Stacks/Stack.cs
using System;
using System.Collections;

// Implementing a stack is trivial using a dynamic array

public class Stack{
    ArrayList stack = new ArrayList();
    public Stack(){}

    public void Push(int n){
        stack.Add(n);
    }
    public int Pop(){
        if(Size() > 0){
            int element = (int)stack[Size() - 1];
            stack.RemoveAt(Size() - 1);
            return element;
        }
        return -1;
    }
    public int Peek(){
        if(Size() > 0){
            return (int)stack[Size() - 1];
        }
        return -1;
    }
    public int Size(){
        return stack.Count;
    }
}

File: Stacks/stack.py
from typing import Any

class Stack:
    def __init__(self):
        self.stack = []
    def push(self, n:Any):
        self.stack.append(n)
    def pop(self)->Any:
        return self.stack.pop()
    def peek(self)->Any:
        return self.stack[-1]